lib/chef/knife/bootstrap/bootstrapper.rb

```diff
@@ -331,7 +331,9 @@ def get_chef_extension_private_params
           pri_config = Hash.new
 
           # validator less bootstrap support for bootstrap protocol cloud-api
-          if (Chef::Config[:validation_key] && !File.exist?(File.expand_path(Chef::Config[:validation_key])))
+          if Chef::Config[:validation_key] && File.exist?(File.expand_path(Chef::Config[:validation_key]))
+            pri_config[:validation_key] = File.read(Chef::Config[:validation_key])
```

Comment:
I think we should `File.read(File.expand_path("Chef::Config[:validation_key]))` here. Otherwise there's a bug where if `Chef::Config[:validation_key]` is `~/.chef/something.pem` then `File.exist?` will return true but `File.read` would raise `Errno::ENOENT`.

```
2.5.1 :001 > File.exist?("~/home.openvpn.conf")
 => false 
2.5.1 :002 > File.exist?(File.expand_path("~/home.openvpn.conf"))
 => true 
2.5.1 :003 > File.read("~/home.openvpn.conf")
Traceback (most recent call last):
        3: from /home/btm/.rvm/rubies/ruby-2.5.1/bin/irb:11:in `<main>'
        2: from (irb):3
        1: from (irb):3:in `read'
Errno::ENOENT (No such file or directory @ rb_sysopen - ~/home.openvpn.conf)
2.5.1 :004 > File.read(File.expand_path("~/home.openvpn.conf"))
 => "snip valid content" 
```